cmake/Platform/Common/Install_common.cmake

```diff
@@ -493,7 +493,7 @@ endfunction()
 function(ly_setup_others)
 
     # List of directories we want to install relative to engine root
-    set(DIRECTORIES_TO_INSTALL Tools/LyTestTools Tools/RemoteConsole)
+    set(DIRECTORIES_TO_INSTALL Tools/LyTestTools Tools/RemoteConsole Gems/AWSCore/Code/Tools/ResourceMappingTool)
```

Comment:
As this is an optional tool, why does it need to be installed here?

Reply:
This should not be installed in the `Install_common.cmake`

This file is for copying general files into the Install layout needed to use O3DE, not a files for a specific gem.

For the AWSCore gem if it wants to copy over a specific file to the install layout it should invoke the `ly_add_install_paths` function defined at the bottom of the file within the AWSCore gem CMakeLists.txt file

Reply:
I'm wondering whether we consider it as an optional tool actually. It may also have some influence on UX decisions like when we want to create the resource mapping config file and awscoreconfiguration.setreg automatically. Our documents seem not providing instructions to manually import resources without this tool.

Reply:
This tool is not an optional tool to editor.
Will try with ly_add_install_paths, last time when I check with @amznestebanpapp Install_common.cmake is the place to add this change

Reply:
The Tools is optional due to by definition being part of a Gem.
The AWSCore gem should responsible for installing all of it's dependencies, not the O3DE engine general.
If a team or studio customizes their engine to not include the AWSCore Gem and then made an install layout, the copy here would fail

Reply:
ideally it should go into gem cmake, but install setup is still in progress which is not finalized. so put in install_common.cmake helps platform team to collect data and define a better pattern.

The downside to put setup in AWSCore gem cmake is we need to remember update the cmake command correspondingly once install setup is finalized. 

I am going to add in our gem cmake but not using `ly_add_install_paths` as it is not working. Synced offline with @lumberyard-employee-dm I will use builtin `install` right now, and we (or platform team) need to update it later.

Reply:
yeah, we need to do a cleanup and move all these "extra files" to the proper place